src/main/java/racingcar/model/utils/common/StringUtils.java

```diff
@@ -0,0 +1,14 @@
+package racingcar.model.utils.common;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class StringUtils {
+
+    private static final String DELIMITER = ",";
+
+    public static List<String> splitStr(final String userInput) {
+        return Arrays.stream(userInput.split(DELIMITER)).collect(Collectors.toList());
```

Comment:
Stream을 사용하지 않아도 될 것 같아요~!
```suggestion
        return Arrays.asList(userInput.split(DELIMITER));
```

Reply:
위 말씀주신 내용의 요점은 Collection을 통한 연산처리의 경우 매 처리 결과마다 Collection을 만들어내기 때문에 메모리를 많이 잡아먹는다는 것을 이야기 하고 싶은 것 같아요! 그에 비해 Stream은 매 처리 결과마다 결과값을 만들어내지 않기 때문에 상대적으로 메모리 사용량이 적다라는 걸 이야기 하고 싶은 것 같구요!!

아래 글들을 읽어보시면 좋을 것 같아요!
[Java 8 Stream vs Collection Storage
](https://stackoverflow.com/questions/29787684/java-8-stream-vs-collection-storage)
[https://jypthemiracle.medium.com/java-stream-api%EB%8A%94-%EC%99%9C-for-loop%EB%B3%B4%EB%8B%A4-%EB%8A%90%EB%A6%B4%EA%B9%8C-50dec4b9974b](https://jypthemiracle.medium.com/java-stream-api%EB%8A%94-%EC%99%9C-for-loop%EB%B3%B4%EB%8B%A4-%EB%8A%90%EB%A6%B4%EA%B9%8C-50dec4b9974b)